go/types/blob.go

```diff
@@ -295,6 +295,7 @@ func readBlob(r io.Reader, vrw ValueReadWriter) Blob {
 	mtChan := make(chan chan metaTuple, runtime.NumCPU())
 
 	makeChunk := func() {
+		rv.Reset()
```

Comment:
Thinking about this more, I thought the reason for the old behavior was so that changes in one chunk don't propagate very far into downstream chunks.